lms/djangoapps/certificates/tests/test_generation_handler.py

```diff
@@ -355,7 +405,7 @@ def test_handle_not_passing_id_verified_cert(self):
             user=different_user,
             course_id=self.course_run_key,
             is_active=True,
-            mode=GeneratedCertificate.MODES.verified,
+            mode=CourseMode.VERIFIED,
```

Comment:
[Kudos] If this is to simplify the number of constants we use in the system, then thank you.